templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+<h1> Delete Car</h1>
+  <p>Man , are u sure u want delete {{ car }} ?</p>
+  <form action="" method="post">
+  {% csrf_token %}
+    <input type="submit" value="Delete!" class="btn btn-danger">
+  </form>
+
+{% endblock %}
```

Comment:
Add new line at the end, also add buttom to cancel deleting and redirecting to previous page.